sources/assets/Stride.Core.Assets.CompilerApp/build/Stride.Core.Assets.CompilerApp.targets

```diff
@@ -160,6 +160,11 @@
   </Target>
 
   <!-- Clean assets -->
```

Comment:
I think always cleaning all assets on clean is too heavy. Sometimes you need to call `dotnet clean` because of a code issue (for instance because of package dependency updates).

I'm ok with a full clean being the default, by I feel like we should have a `SkipAssetsClean` option.

Reply:
Added a condition, let me know